### PR TITLE
Avoid role/tasks tag collision

### PR DESCRIPTION
**Pull request by OmarIthawi**
Configuration Pull Request
---

### Problem
When running with `--tags=letsencrypt` it runs the whole playbook and it's dependencies, where's I want it to run only the tasks in the playbook.

### Solution
Add `_role` prefix to all role tag to avoid collision.

### Review
Make sure that the following steps are done before merging

  - [ ] @devops team member has commented with :+1:
  - [ ] are you adding any new default values that need to be overridden when this goes live?  
    - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by OmarIthawi** [comment]
@mrgnr and @melvinsoft could you please take a look? Please assign someone else if you're not free.

**Review by melvinsoft — COMMENTED** [review]
👍 LTBM

**Comment by OmarIthawi** [comment]
Thanks @melvinsoft!

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/edx_notes_api.j2 [hunk 1]
```diff
@@ -1,3 +1,5 @@
+{% if INSTALL_EDX_NOTES %}
```
Comment (tkeemon) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/edx_notes_api.j2]: I don't understand this change. edx-notes will only be installed if `INSTALL_EDX_NOTES` is set to `True` and this template will only be included if `edx_notes_api` is in the `nginx_sites` list variable. 

Does this conditional serve any additional purpose?

**Review by tkeemon — COMMENTED** [review]
The changes look good to me. I'll defer to @mrgnr though.

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/edx_notes_api.j2 (hunk 1 above)
Comment (OmarIthawi) [reply playbooks/roles/nginx/templates/edx/app/nginx/sites-available/edx_notes_api.j2]: @tkeemon You're right it shouldn't be included. However by default [`INSTALL_EDX_NOTES` isn't defined so the role won't be installed](https://github.com/appsembler/configuration/blob/b80f1b9/playbooks/appsemblerPlaybooks/ficus_basic.yml#L51-L53).

However the [`edx_notes_api` is included by default in the nginx roles](https://github.com/appsembler/configuration/blob/b80f1b9/playbooks/appsemblerPlaybooks/ficus_basic.yml#L30-L35) all the time.

While sometimes this issue passes silently without explicit errors, the nginx does indeed make it into the server and creates errors (in specific scenarios).

Deleting the file manually might have done it for me in my very specific scenario, which I don't clearly remember, but the `edx_notes_api` shouldn't make it to the server anyway.

**Review by tkeemon — APPROVED** [review]
These changes look. Let's get this merged.

**Comment by OmarIthawi** [comment]
Thanks a lot @tkeemon! Merging.

**Pull request merged**